service/src/main/kotlin/fi/espoo/evaka/messaging/daycarydailynote/DaycareDailyNoteController.kt

```diff
@@ -85,6 +85,36 @@ class DaycareDailyNoteController(
         return db.transaction { ResponseEntity.ok(it.updateDaycareDailyNote(body.copy(childId = childId))) }
     }
 
+    @PostMapping("/group/{groupId}")
+    fun createDailyNoteForGroup(
+        db: Database.Connection,
+        user: AuthenticatedUser,
+        @PathVariable groupId: UUID,
+        @RequestBody body: DaycareDailyNote
+    ): ResponseEntity<UUID> {
+        Audit.DaycareDailyNoteCreate.log(user.id)
```

Comment:
Nit (as not changed here): the audit event should probably have an `objectId` too to known what the note was created for.